src/AppBundle/Helper/ApiHelper.php

```diff
@@ -50,6 +51,7 @@ public function groups($project, $username)
             if (isset($res["batchcomplete"]) && isset($res["query"]["users"][0]["groups"])) {
                 $result = $res["query"]["users"][0]["groups"];
             }
+            $this->cacheSave($cacheKey, $result, 'P7D');
```

Comment:
I figured anything that reduces the number of API calls is good, but I'm happy either way. Will remove; good to keep it simple anyway. :)